src/cpp/ext/csm/metadata_exchange.cc

```diff
@@ -493,9 +498,13 @@ bool ServiceMeshLabelsInjector::AddOptionalLabels(
 }
 
 size_t ServiceMeshLabelsInjector::GetOptionalLabelsSize(
+    bool is_client,
     absl::Span<const std::shared_ptr<std::map<std::string, std::string>>>)
     const {
-  return 2;
+  if (is_client) {
```

Comment:
Can write this as `return is_client ? 2 : 0`.

Reply:
Done.